colors/default.kak

```diff
@@ -1,50 +1,57 @@
 # Kakoune default color scheme
+#
+# Template: https://github.com/mawww/kakoune/blob/master/colors/default.kak
+# Documentation: https://github.com/mawww/kakoune/blob/master/doc/pages/faces.asciidoc
```

Comment:
I would prefer no references to github in the codebase, I cannot see any good reason to direct people towards the WWW version of files that they already have on their filesystem.

Reply:
It was for people browsing the file and persons writing color schemes.

I find having the URLs at hand make it easy to notice changes and where the custom color schemes come from.

Reply:
I still dont think the code should refer to wherever my Kakoune git repository happens to be hosted at the moment, I´d like to keep as low as possible coupling with github.